docs/tutorials/demo/Alchemiscale Demo.ipynb

```diff
@@ -0,0 +1,2774 @@
+    "\n",
+    "The `Protocol` defines the actual computational method used to evaluate the `Transformation` to yield estimates for the free energy difference between the `ChemicalSystem`s.\n",
+    "\n",
+    "The `ComponentMapping` defines the atom mapping(s) between corresponding `Component`s in the two `ChemicalSystem`s. This is often critical for relative binding free energy calculations, since the choice of mapping can heavily influence convergence of the resulting estimates."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fae8b80e-1b77-4d64-a967-10f75326647c",
+   "metadata": {},
+   "source": [
+    "#### Define the `Protocol` used for `Transformation` evaluation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2101fea6-3fda-4a73-81a0-cea41bb30dc0",
+   "metadata": {},
+   "source": [
+    "For this example, we'll use the same `Protocol` for all our `Transformation`s, with identical `Settings` for each."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "183b23a6-1839-4586-bbbe-60b2defd2f01",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from openfe.protocols.openmm_rfe import RelativeHybridTopologyProtocol"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fb216611-780c-447b-b9d6-b628ba9ea619",
+   "metadata": {},
+   "source": [
+    "Any given `Protocol` has a `default_settings()` method, which can be used to get the default settings that are specific to that `Protocol`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "d7bac455-ddc1-4735-96d7-5c6681ab3cbc",
+   "metadata": {
+    "scrolled": true,
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'forcefield_settings': {'constraints': 'hbonds',\n",
+       "  'rigid_water': True,\n",
+       "  'remove_com': False,\n",
+       "  'hydrogen_mass': 3.0,\n",
+       "  'forcefields': ['amber/ff14SB.xml',\n",
+       "   'amber/tip3p_standard.xml',\n",
+       "   'amber/tip3p_HFE_multivalent.xml',\n",
+       "   'amber/phosaa10.xml'],\n",
+       "  'small_molecule_forcefield': 'openff-2.0.0'},\n",
+       " 'thermo_settings': {'temperature': 298.15 <Unit('kelvin')>,\n",
+       "  'pressure': 0.9869232667160129 <Unit('standard_atmosphere')>,\n",
+       "  'ph': None,\n",
+       "  'redox_potential': None},\n",
+       " 'system_settings': {'nonbonded_method': 'PME',\n",
+       "  'nonbonded_cutoff': 1.0 <Unit('nanometer')>},\n",
+       " 'solvation_settings': {'solvent_model': 'tip3p',\n",
+       "  'solvent_padding': 1.2 <Unit('nanometer')>},\n",
+       " 'alchemical_settings': {'lambda_functions': 'default',\n",
+       "  'lambda_windows': 11,\n",
+       "  'unsampled_endstates': False,\n",
+       "  'use_dispersion_correction': False,\n",
+       "  'softcore_LJ_v2': True,\n",
+       "  'softcore_electrostatics': True,\n",
```

Comment:
I believe the softcore_electrostatics entry have been removed now as of ~ openfe 0.12?

Reply:
Fair enough; I think I should show the versions of `openfe`, `gufe`, and `alchemiscale` used for the notebook generation. I'll add these.

Reply:
Done!